samples/ai-image-tagging/src/ImageTaggingHelp/ImageTaggingHelp.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List, ListItem, Paragraph } from '@contentful/forma-36-react-components';
+
+import './ImageTaggingHelp.css'
+
+const getContentTypeUrl = (contentType) => {
+  const spaceId = contentType.sys.space.sys.id;
+  const contentTypeId = contentType.sys.id;
+
+  return `https://app.contentful.com/spaces/${spaceId}/content_types/${contentTypeId}/fields`
+};
+
+export class ImageTaggingHelp extends React.Component {
+  static propTypes = {
+    contentType: PropTypes.object.isRequired,
+    hasImageField: PropTypes.bool.isRequired,
+    hasTagField: PropTypes.bool.isRequired
+  };
+
+  render = () => {
+    const { hasImageField, hasTagField, contentType } = this.props;
+
+    return (
+      <div>
+        <Paragraph element='p' extraClassNames={'f36-color--text-light'}>
+          To be able to use image auto tagging, add the following fields to your content type <a
```

Comment:
I'll check if we can easily access that info in the extension

It could say something like 'Ask your content model administrator to resolve the following issues with the extension configuration: ...'

Reply:
As discussed, not adding this for now